pkg/runtime/interfaces/application_configuration.go

```diff
@@ -18,6 +21,9 @@ type DbConfig struct {
 	DeprecatedDebug        bool   `json:"debug" pflag:",deprecated"`
 
 	EnableForeignKeyConstraintWhenMigrating bool           `json:"enableForeignKeyConstraintWhenMigrating" pflag:",Whether to enable gorm foreign keys when migrating the db"`
+	MaxIdleConnections                      int            `json:"maxIdleConnections" pflag:",maxIdleConns sets the maximum number of connections in the idle connection pool."`
+	MaxOpenConnections                      int            `json:"maxOpenConnections" pflag:",maxOpenConns sets the maximum number of open connections to the database."`
+	ConnMaxLifeTime                         time.Duration  `json:"connMaxLifeTime" pflag:",sets the maximum amount of time a connection may be reused"`
```

Comment:
I think you have to use config.Duration to get the right unmarshaling behavior from commandline...

```suggestion
	ConnMaxLifeTime                         config.Duration  `json:"connMaxLifeTime" pflag:",sets the maximum amount of time a connection may be reused"`
```